tmt/steps/report/junit.py

```diff
@@ -54,6 +55,22 @@ def duration_to_seconds(duration: Optional[str]) -> Optional[int]:
             f"Malformed duration '{duration}' ({error}).")
 
 
+def stdout_filter(log: str, msg_type: str = 'FAIL') -> str:
+    """ Filter stdout and get only messages with certain type """
+    filtered = ''
+
+    # Filter beakerlib style logs
+    for m in re.findall(f'.*\\[\\s+{msg_type}\\s+\\].*', log, re.MULTILINE):
```

Comment:
`f` and `r` strings can be combined for fewer backslashes: `fr'.*\[\s+{msg_type}\s+\].*'`